agr_java_core/src/main/java/org/alliancegenome/neo4j/view/View.java

```diff
@@ -28,6 +28,7 @@
     public static class GeneAPI extends API { }
     public static class DiseaseAPI extends API { }
     public static class GeneAllelesAPI extends API { }
+    public static class GeneAlleleVariantSequenceAPI extends Default { }
```

Comment:
@cmpich this should extend API not Default?

Reply:
No, this is intentional at this moment. I am trying to optimize the output, i.e. reduce the payload to only the attributes really needed I am extending Default.